docs/creating_component_deps_and_reqs.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: ""
+title: "Moved Creating Integration Manifest"
```

Comment:
I feel like we should just delete these pages and add redirects

Reply:
Makes sense.